packages/block-library/src/categories/index.php

```diff
@@ -31,7 +31,8 @@ function render_block_core_categories( $attributes ) {
 		$wrapper_markup           = '<div class="%1$s">%2$s</div>';
 		$items_markup             = wp_dropdown_categories( $args );
 		$type                     = 'dropdown';
-
+		$title                    = __( 'Categories', 'gutenberg' );
+		
```

Comment:
It looks like there is a whitespace here that is making the PHP linter unhappy? See https://travis-ci.org/WordPress/gutenberg/jobs/447997322

Reply:
Thanks, I can make this fix real quick!